Remove shh and shhnext from APIModules

adambabik opened the pull request:
Changes:
- merged again https://github.com/status-im/status-go/pull/993 which was reverted due to issues in status-react

mandrigin reviewed (approved):
LGTM, but... 5 commits? :)

adambabik commented:
@mandrigin I did not squash them but they will be in `develop`.

adambabik committed:
Remove shh and shhext from APIModules

Addresses #912. By removing these two modules from APIModules, we disallow dapps from using dangerous methods like shh.SetMinPoW

adambabik committed:
fix e2e tests after removing shh API modules
